### DISCUSSION TITLE
Scene can't be imported anymore

### DISCUSSION THREAD

**Discussion by oli414** [mozilla/hubs#3869] [Hubs Cloud Questions] [answered]
We wanted to update one of our scene in our hubs cloud instance, however all of the sudden we get an error when importing (in the console) and the loading icon stays.

The error we see is:
400 Bad Request

I'm importing the scene from the official mozilla hubs site (instead of our own hubs cloud instance) and everything was working fine up until now.

I tried importing some different scenes, but they seem to import without any issues. Additionally I tried importing earlier versions (That were copied and thus have a different ID) of the scene that no longer works, and these too don't import, even though they previousily have been used on production without any changes.
So it appears to be tied to the scene, given the circumstances the only thing I can think of is that there might be something wrong with one of the resources that the scene relies on. But I'm not sure.

Another thing is that we are running another hubs cloud instance, but it looks like we can upload the scene there without issue. I just updated the hubs cloud client as well to the latest version but that didn't fix the issue.

What can I do to get this scene back up and running?

**Just to sum it up:**
- Scene doesn't import in one of our hubs cloud instances (400 bad request in the console)
- Other scenes can be imported just fine through the same workflow
- Previous iterations of the scene suddenly don't import either, even though they have been imported previousily without issue
- Hubs cloud is running the latest version of the client
- Scene works on a different hubs cloud instance
- Scene works on the official Mozilla Hubs Spoke
- Scene works on the official Mozilla Hubs when used to create a room

I can only share the scene URL in private.

**Comment by robinkwilson** [comment, marked as answer]
Thank you for the Summary at the end! Helps me a lot. 

What external assets is that scene reliant on? 
I'm not sure if it was a `Bad Request` error, but another user had to delete each asset individually to find the broken asset. Once that asset was removed from the scene, they could access/import the scene successfully again.

I know it sounds arduous, but I'd look through the assets to find any broken urls then remove that asset so you can access the scene. 

Let us know how it goes!

**Reply by robinkwilson** [reply]
Marking as answered for now, please unmark as answered if you need more help!
